src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const husbandOrWife = person.isMarried
+    ? `${person.partnerName} is my ${person.sex === 'm'
+      ? 'wife' : 'husband'}` : 'I am not married';
+  const ageContant = person.age
+    ? <p className="Person__age">{`I am ${person.age}`}</p> : '';
+
+  return (
+    <section className="Person">
+      <>
+        <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+        {ageContant}
```

Comment:
```suggestion
        { age && <p className="Person__age">{`I am ${person.age}`}</p> }
```

Reply:
Hi, I don't understand what I need to do